docs/content/en/docs/Installation/netboot.md

```diff
@@ -21,10 +21,10 @@ Booting using these files can happen in two ways:
 - Software based network booting. This works with a special ISO, built with
   [ipxe](https://ipxe.org/) project. Kairos releases include pre-built ISOs for
   netbooting (named like `*.ipxe.iso.ipxe`).
-
+- Use [AuroraBoot](/docs/reference/auroraboot)
```

Comment:
By adding this option, we need to change line 18 to say "three ways" or "multiple ways" better